api/src/main/java/org/openmrs/api/db/PatientDAO.java

```diff
@@ -130,8 +133,18 @@ public interface PatientDAO {
 	 */
 	public List<Patient> getPatients(String name, List<PatientIdentifierType> identifierTypes,
 		boolean matchIdentifierExactly, Integer start, Integer length) throws DAOException;
-	
-	
+
+	/**
+	 * Get patient by universally unique identifier.
+	 * 
+	 * @param uuid universally unique identifier
+	 * @throws APIException
+	 * <strong>Should</strong> fetch patient with given identifier
+	 * <strong>Should</strong> return null if patient not found with given identifier
+	 */
+	@Authorized( { PrivilegeConstants.GET_PATIENTS })
```

Comment:
I don't believe the Authorized annotation should be in the PatientDAO interface.  This is only used in PatientService interface.